### ISSUE TITLE
I cannot use cmder with a screen reader 

### ISSUE THREAD

**Issue by MesterPerfect** [cmderdev/cmder#2495]
### Purpose of the issue
- [ ] Bug report 

### Version Information
cmder_mini 1.3.17 (2020-12-23)

Windows 8.1 Pro

### Description of the issue
am blind user and use a screen reader called nvda
When I type commands in cmder or delete them nothing is read from my screen reader which makes me unable to make sure that the commands I writing are correct.

**Comment by daxgames** [comment]
Cmder is a bundle of 4 other products, plus some scripts to glue them together.

The settings UI referred to above is part of ConEmu.

Cmder the glue that makes the other 4 products (ConEmu is the console window, Clink is an extension to CMD, clink-completions is a set of scripts to enhance Clink, and git for Windows is the fourth product) work together.

This issue report/feature request is probably for Conemu.  You can report that [here](https://github.com/maximus5/conemu)
